subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -554,6 +554,76 @@ static void sw_switch(uint8_t dir, uint8_t phy_curr, uint8_t flags_curr, uint8_t
 	sw_tifs_toggle += 1U;
 	sw_tifs_toggle &= 1;
 }
+
+static void sw_switch_pback(uint8_t phy, uint8_t flags)
```

Comment:
Something that I am trying to use for BIS subevents: https://github.com/zephyrproject-rtos/zephyr/pull/33118

Reply:
`sw_switch` is not inlined so it will not be optimized (unless I misunderstood what you meant) for new case, but regardless reusing `sw_switch` as in mentioned PR works for me since it's just a single extra branch so not much of a difference. I initially thought that there will be more to change there thus I created a copy.